x11rb/src/lib.rs

```diff
@@ -205,3 +206,20 @@ pub const CURRENT_TIME: Timestamp = 0;
 
 /// This constant can be used to fill unused entries in `Keysym` tables
 pub const NO_SYMBOL: Keysym = 0;
+
+#[cfg(windows)]
```

Comment:
Perhaps this should become `not(unix)` here and in `Cargo.toml` (dunno if `Cargo.toml` supports that).

If trying to connect to an X11 server just panics, that basically makes the whole crate unusable. So, I think I'd prefer a compile error.

Reply:
I've changed it to use `not(unix)`